### PR TITLE
Fixes my install problem.

### PR DESCRIPTION
**Pull request by philipn**
To reproduce:

```
  git clone <this repo>

  virtualenv env
  source env/bin/activate
  python setup.py install
  cd ~/ (or anywhere the module won't be on the current path)

  python

  > import armstrong.esi
```

Will fail, as the `__init__.py` isn't installed.

I think you all know this namespace packaging stuff better than I do, so please let me know if there's a better solution here!

### CODE REVIEW

**Comment by joncotton** [comment]
TLDR: `pip install .` to install the locally checked out repo. Don't use "python setup.py".

This happens because the setup.py file is lying to you when it imports setup() from distutils. Armstrong is a namespaced package and distutils can't handle those. I mean we've tried, but it's so very very not worth it, especially now that Python has standardized on Pip and Setuptools. We are updating Armstrong components to stop lying and use better packaging (e.g. the relevant update to [ArmSections](https://github.com/armstrong/armstrong.core.arm_sections/commit/ef308a70c6734d4912d21e566a7de2e7b2f2f5f6)). When you install with Pip however, Pip uses Setuptools instead of distutils automatically behind the scenes.

Given the necessary Setuptools conversion anyway, I'm going to reject this PR. But definitely try installing with pip and let me know if you have trouble.

**Comment by philipn** [comment]
Unfortunately, there doesn't appear to be a way to install this via a install_requires and a custom package URL (see https://github.com/localwiki/localwiki/blob/master/setup.py for an example) in our setup.py. The same problem presents itself there.

**Comment by joncotton** [comment]
So when you put "armstrong.esi" in `install_requires` and add the Github link to your modified fork in `dependency_links`, it fails? Are you using "python setup.py install" or pip to install localwiki? I'd expect pip to work.

Since you have your own fork anyway, replace the distutils import with `from setuptools import setup` in setup.py. Try that with "python setup.py install".

Lastly, I'll call out @crccheck at the Texas Tribune. TT wrote this component and may be curious about your HTTP_HOST header addition. Maybe mainline should have that too or a flexible way to manipulate other headers.
